spec/unit/resource/chocolatey_installer_spec.rb

```diff
@@ -0,0 +1,131 @@
+#
+# Copyright:: Copyright (c) Chef Software Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require "spec_helper"
+
+describe Chef::Resource::ChocolateyInstaller do
+  include RecipeDSLHelper
+
+  let(:resource) { Chef::Resource::ChocolateyInstaller.new("fakey_fakerton") }
+  let(:config) do
+    <<-CONFIG
```

Comment:
Maybe it's guilt over my own offenses, but I get a bit twitchy with non-trivial file content is embedded in the spec itself. Could this be broken out into a "fixture" XML instead?